app/views/katello/api/v2/repositories/show.json.rabl

```diff
@@ -47,7 +47,3 @@ if @object && @object.library_instance_id.nil?
   end
 
 end
-
-child :environment => :environment do |_repo|
-  attribute :id
-end
```

Comment:
Oh, I believe this was because prior to my change above, it would return you repositories across ALL environments without any indication on the repository (since its an index call) of which environment each repo was in. This is likely not necessary after the change in the controller.